test/filter-tests.js

```diff
@@ -95,14 +95,10 @@ function testFilter(filename, suffix, filters) {
 // bless("cases/gcc-arm-sum.asm", "cases/gcc-arm-sum.asm.directives.labels.comments.json", {directives: true, labels: true, commentOnly: true});
```

Comment:
You have spotted my terrible low-tech "solution" to "I changed the code and now the tests are broken..."

Reply:
can I just delete these lines?